trillian/integration/ct_integration_test.cfg

```diff
@@ -1,26 +1,39 @@
-[
-	{
-		"LogID": @TREE_ID@,
-		"Prefix": "athos",
-		"RootsPEMFile": ["@TESTDATA@/fake-ca.cert"],
-		"PubKeyPEMFile": "@TESTDATA@/ct-http-server.pubkey.pem",
-		"PrivKeyPEMFile": "@TESTDATA@/ct-http-server.privkey.pem",
-		"PrivKeyPassword": "dirk"
+{
+	"log_id": @TREE_ID@,
+	"prefix": "athos",
+	"roots_pem_file": ["@TESTDATA@/fake-ca.cert"],
+	"public_key": {
+		"der": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEB/hRr6qMVoOQMbeA49Ya9y82BnHs3Tu+fjZvDRwcYAt/9Z//5SRJNFbySxBfvwgf+Q7PNbWKioswClS3vx1NuQ=="
```

Comment:
Be good to add a comment that explains the provenance of the binary data.  E.g. something like "Created by running `openssl -in testdata/ct-http-server.pubkey.pem -outform der | base64`" (or whatever).

(If JSON commenting causes a problem, then adding to the commit message is probably good enough, so it can be found with `git blame`)